README.md

```diff
@@ -9,6 +9,14 @@ An editor for Bayesian Networks built in React that uses Bayes' theorem.
 
 In [probability theory](https://en.wikipedia.org/wiki/Probability_theory) and [statistics](https://en.wikipedia.org/wiki/Statistics), Bayes’ theorem describes the probability of an event, based on prior knowledge of conditions that might be related to the event. For example, if cancer is related to age, then, using Bayes’ theorem, a person's age can be used to more accurately assess the probability that they have cancer than can be done without knowledge of the person’s age.
 
+Bayes' Theorem is Mathematically stated as the following equation:
+P(A|B) = (P(B|A).P(A)) / P(B)
+
+where A and B are events and P(B)!= 0
+
+* P(A) is a conditional probability: the likelihood of event A occurring given that B is true.
```

Comment:
Nice contribution, but you should update `P(A)` to `P(A | B)` and add a link to `conditional probability` 😄

```
...is a [conditional probability](https://en.wikipedia.org/wiki/Conditional_probability): the...
```
 
Also, you could add:

```
P(B | A) is also a conditional probability: the likelihood of event B occurring given that A is true.
P(A) and P(B) are the probabilities of observing A and B independently of each other; this is known as the [marginal probability](https://en.wikipedia.org/wiki/Marginal_distribution).
```

Reply:
Done.